src/core/issues-notes.js

```diff
@@ -23,7 +23,18 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    Array.from($ins)
+      .filter(({ dataset: { number } }) => number && !ghIssues[number] && conf.githubAPI)
```

Comment:
I have added an extra test for `conf.githubAPI` as without this two-three tests fail. The reason being that there are cases where user sets up an issue base that is not related to any github repo so in that case `ghIssues[number]` by default goes to `undefined` and thus leading to removal of that issue from the spec. I found this out when I was analysing the tests located inside https://github.com/w3c/respec/blob/develop/tests/spec/core/issues-notes-spec.js and an example can be given for https://github.com/w3c/respec/blob/3e505449ad98fbc3e6977de1c976fbf05df5dc38/tests/spec/core/issues-notes-spec.js#L181

Reply:
@saschanaz you can now have a look. Shall I begin writing tests for this ?

Reply:
It does that... remember, you can test everything locally (see dev guide). Don't rely on Travis during development as it can be very slow.

Reply:
Yep that is how I test using localhost:8080/tests/specRunner.html

Reply:
So shall I write tests now. If everything is ok

Reply:
Tests would be great!

Reply:
>I have added an extra test for `conf.githubAPI` as without this two-three tests fail.

So it means the filter requires `conf.githubAPI`. In that case maybe we should add if-statement to skip the filtering itself if not needed..